# HG changeset patch
# User Andreas Weber <andy.weber.aw@gmail.com>
# Date 1427990087 -7200
#      Thu Apr 02 17:54:47 2015 +0200
# Node ID ab825e51dc687f886110ff83b770a95751318c0a
# Parent  b66ab1af5a3be999c0cab5857a4cd43ff557395a
de_DE.ts: Finish german translation of GUI strings

* de_DE.ts: Merge translations from Armin Müller and
  David Rörich, now complete.

* contributors.in: Add David Rörich

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -277,16 +277,17 @@ Sander van Rijn
 Petter Risholm
 Matthew W. Roberts
 Dmitry Roshchin
 Peter Rosin
 Andrew Ross
 Fabio Rossi
 Mark van Rossum
 Joe Rothweiler
+David Rörich
 Kevin Ruland
 Kristian Rumberg
 Ryan Rusaw
 Olli Saarela
 Toni Saarela
 Juhani Saastamoinen
 Radek Salac
 Mike Sander
diff --git a/libgui/languages/de_DE.ts b/libgui/languages/de_DE.ts
--- a/libgui/languages/de_DE.ts
+++ b/libgui/languages/de_DE.ts
@@ -91,99 +91,132 @@
         <location filename="../qterminal/libqterminal/win32/QWinTerminalImpl.cpp" line="+1778"/>
         <source>copied selection to clipboard</source>
         <translation>Auswahl in die Zwischenablage kopiert</translation>
     </message>
 </context>
 <context>
     <name>QtHandles::Figure</name>
     <message>
+        <location filename="../graphics/Figure.cc" line="+281"/>
         <source>Figure ToolBar</source>
-        <translation type="obsolete">Werkzeugleiste</translation>
-    </message>
-    <message>
+        <translation>Werkzeugleiste</translation>
+    </message>
+    <message>
+        <location line="+9"/>
+        <source>Axes</source>
+        <translation>Achsen</translation>
+    </message>
+    <message>
+        <location line="+4"/>
+        <source>Grid</source>
+        <translation>Gitter</translation>
+    </message>
+    <message>
+        <location line="+4"/>
+        <source>Autoscale</source>
+        <translation>Autoskalierung</translation>
+    </message>
+    <message>
+        <location line="+7"/>
         <source>&amp;File</source>
-        <translation type="obsolete">&amp;Datei</translation>
-    </message>
-    <message>
+        <translation>&amp;Datei</translation>
+    </message>
+    <message>
+        <location line="+2"/>
         <source>&amp;Save</source>
-        <translation type="obsolete">&amp;Speichern</translation>
-    </message>
-    <message>
+        <translation>&amp;Speichern</translation>
+    </message>
+    <message>
+        <location line="+1"/>
         <source>Save &amp;As</source>
-        <translation type="obsolete">Speichern &amp;unter</translation>
-    </message>
-    <message>
+        <translation>Speichern &amp;unter</translation>
+    </message>
+    <message>
+        <location line="+2"/>
         <source>&amp;Close Figure</source>
-        <translation type="obsolete">Abbildung s&amp;chließen</translation>
-    </message>
-    <message>
+        <translation>Abbildung s&amp;chließen</translation>
+    </message>
+    <message>
+        <location line="+3"/>
         <source>&amp;Edit</source>
-        <translation type="obsolete">&amp;Bearbeiten</translation>
-    </message>
-    <message>
+        <translation>&amp;Bearbeiten</translation>
+    </message>
+    <message>
+        <location line="+2"/>
         <source>Cop&amp;y</source>
-        <translation type="obsolete">&amp;Kopieren</translation>
-    </message>
-    <message>
+        <translation>&amp;Kopieren</translation>
+    </message>
+    <message>
+        <location line="+2"/>
         <source>Cu&amp;t</source>
-        <translation type="obsolete">&amp;Ausschneiden</translation>
-    </message>
-    <message>
+        <translation>&amp;Ausschneiden</translation>
+    </message>
+    <message>
+        <location line="+2"/>
         <source>&amp;Paste</source>
-        <translation type="obsolete">E&amp;infügen</translation>
-    </message>
-    <message>
+        <translation>E&amp;infügen</translation>
+    </message>
+    <message>
+        <location line="+5"/>
         <source>&amp;Help</source>
-        <translation type="obsolete">&amp;Hilfe</translation>
-    </message>
-    <message>
+        <translation>&amp;Hilfe</translation>
+    </message>
+    <message>
+        <location line="+2"/>
         <source>&amp;About QtHandles</source>
-        <translation type="obsolete">&amp;Über QtHandles</translation>
-    </message>
-    <message>
+        <translation>&amp;Über QtHandles</translation>
+    </message>
+    <message>
+        <location line="+2"/>
         <source>About &amp;Qt</source>
-        <translation type="obsolete">Über &amp;Qt</translation>
-    </message>
-    <message>
+        <translation>Über &amp;Qt</translation>
+    </message>
+    <message>
+        <location line="+427"/>
         <source>About QtHandles</source>
-        <translation type="obsolete">Über QtHandles</translation>
-    </message>
-    <message>
+        <translation>Über QtHandles</translation>
+    </message>
+    <message>
+        <location line="+39"/>
         <source>Save Figure As</source>
-        <translation type="obsolete">Abbildung speichern unter</translation>
+        <translation>Abbildung speichern unter</translation>
     </message>
 </context>
 <context>
     <name>QtHandles::MouseModeActionGroup</name>
     <message>
+        <location filename="../graphics/MouseModeActionGroup.cc" line="+40"/>
         <source>Rotate</source>
-        <translation type="obsolete">Rotieren</translation>
-    </message>
-    <message>
+        <translation>Rotieren</translation>
+    </message>
+    <message>
+        <location line="+2"/>
         <source>Zoom In</source>
-        <translatorcomment>David: Removed &apos;Ansicht&apos; as that is the name of the parent menu</translatorcomment>
-        <translation type="obsolete">Vergrößern</translation>
-    </message>
-    <message>
+        <translation>Vergrößern</translation>
+    </message>
+    <message>
+        <location line="+4"/>
         <source>Zoom Out</source>
-        <translatorcomment>David: Removed &apos;Ansicht&apos; as that is the name of the parent menu</translatorcomment>
-        <translation type="obsolete">Verkleinern</translation>
-    </message>
-    <message>
+        <translation>Verkleinern</translation>
+    </message>
+    <message>
+        <location line="+4"/>
         <source>Pan</source>
-        <translation type="obsolete">Schwenken</translation>
-    </message>
-    <message>
+        <translation>Schwenken</translation>
+    </message>
+    <message>
+        <location line="+2"/>
         <source>Insert Text</source>
-        <translation type="obsolete">Text einfügen</translation>
-    </message>
-    <message>
+        <translation>Text einfügen</translation>
+    </message>
+    <message>
+        <location line="+2"/>
         <source>Select</source>
-        <translation type="obsolete">Auswählen</translation>
+        <translation>Auswählen</translation>
     </message>
 </context>
 <context>
     <name>documentation_dock_widget</name>
     <message>
         <location filename="../src/documentation-dock-widget.cc" line="+34"/>
         <source>Documentation</source>
         <translation>Dokumentation</translation>
@@ -240,32 +273,32 @@ ist im Editor bereits geöffnet</translation>
     <message>
         <location line="+388"/>
         <source>&amp;Save File</source>
         <translation>Datei &amp;speichern</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Save File &amp;As...</source>
-        <translation>Datei speichern &amp;als...</translation>
+        <translation>D&amp;atei speichern unter...</translation>
     </message>
     <message>
         <location line="+14"/>
         <source>Print...</source>
         <translation>Drucken...</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>&amp;Undo</source>
         <translation>&amp;Rückgängig</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Redo</source>
-        <translation>&amp;Wiederholen</translation>
+        <translation>&amp;Wiederherstellen</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>&amp;Copy</source>
         <translation>&amp;Kopieren</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -280,22 +313,22 @@ ist im Editor bereits geöffnet</translation>
     <message>
         <location line="+96"/>
         <source>&amp;Next Bookmark</source>
         <translation>&amp;Nächstes Lesezeichen</translation>
     </message>
     <message>
         <location line="-2"/>
         <source>Pre&amp;vious Bookmark</source>
-        <translation>&amp;Voriges Lesezeichen</translation>
+        <translation>&amp;Vorheriges Lesezeichen</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Toggle &amp;Bookmark</source>
-        <translation>&amp;Lesezeichen setzen</translation>
+        <translation>&amp;Lesezeichen umschalten</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Remove All Bookmarks</source>
         <translation>Alle Lesezeichen &amp;entfernen</translation>
     </message>
     <message>
         <location line="+54"/>
@@ -399,17 +432,17 @@ ist im Editor bereits geöffnet</translation>
     <message>
         <location line="+31"/>
         <source>Select All</source>
         <translation>Alles auswählen</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>&amp;Commands</source>
-        <translation>Befehle</translation>
+        <translation>Befe&amp;hle</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Delete Line</source>
         <translation>Zeile löschen</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -522,34 +555,34 @@ ist im Editor bereits geöffnet</translation>
         <translation>&amp;Editor</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Show &amp;Line Numbers</source>
         <translation>Zei&amp;lennummern anzeigen</translation>
     </message>
     <message>
-        <location line="+4"/>
-        <source>Show &amp;White Spaces</source>
-        <translation>Leerzeichen anzeigen</translation>
-    </message>
-    <message>
-        <location line="+4"/>
+        <location line="+128"/>
+        <source>Editor</source>
+        <translation>Editor</translation>
+    </message>
+    <message>
+        <location line="-120"/>
         <source>Show Line &amp;Endings</source>
-        <translation>Zeilenenden anzeigen</translation>
+        <translation>Z&amp;eilenenden anzeigen</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Show &amp;Indentation Guides</source>
-        <translation>Einrückungshinweise anzeigen</translation>
+        <translation>E&amp;inrückungshinweise anzeigen</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Show Long Line &amp;Marker</source>
-        <translation>Markierung langer Zeilen anzeigen</translation>
+        <translation>&amp;Markierung langer Zeilen anzeigen</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Zoom &amp;In</source>
         <translation>Ver&amp;größern</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -621,17 +654,22 @@ konnte nicht zum Schreiben geöffnet werden:
         <translation>&amp;Datei</translation>
     </message>
     <message>
         <location line="+36"/>
         <source>&amp;Edit</source>
         <translation>&amp;Bearbeiten</translation>
     </message>
     <message>
-        <location line="+166"/>
+        <location line="+138"/>
+        <source>Show &amp;Whitespace Characters</source>
+        <translation>Lee&amp;rzeichen anzeigen</translation>
+    </message>
+    <message>
+        <location line="+28"/>
         <source>&amp;Debug</source>
         <translation>&amp;Debuggen</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>&amp;Run</source>
         <translation>&amp;Ausführen</translation>
     </message>
@@ -755,17 +793,17 @@ call your script using its name as an Oc
 
 Do you want to choose another name?</source>
         <translation>&quot;%1&quot;
 ist kein gültiger Bezeichner.
 
 Wird das Skript unter diesem Namen gespeichert, kann
 es nicht als Octave-Befehl aufgerufen werden.
 
-Möchten Sie einen anderen anderen Namen wählen?</translation>
+Einen anderen Namen wählen?</translation>
     </message>
     <message>
         <location line="+85"/>
         <source>It seems that &apos;%1&apos; has been modified by another application. Do you want to reload it?</source>
         <translation>Die Datei %1 wurde von einer anderen Anwendung verändert. Soll der neue Inhalt geladen werden?</translation>
     </message>
 </context>
 <context>
@@ -781,17 +819,22 @@ Möchten Sie einen anderen anderen Namen wählen?</translation>
         <translation>Dateien durchsuchen.</translation>
     </message>
     <message>
         <location line="+18"/>
         <source>Enter the path or filename</source>
         <translation>Pfad oder Dateinamen eingeben</translation>
     </message>
     <message>
-        <location line="+28"/>
+        <location line="+24"/>
+        <source>Set Octave directory to current browser directory</source>
+        <translation>Setze Octave Verzeichnis zu aktuellem Browser Verzeichnis</translation>
+    </message>
+    <message>
+        <location line="+4"/>
         <source>Actions on current directory</source>
         <translation>Aktionen mit aktuellem Verzeichnis</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>Set Browser Directory...</source>
         <translation>Wechsle in Verzeichnis...</translation>
     </message>
@@ -822,22 +865,17 @@ Möchten Sie einen anderen anderen Namen wählen?</translation>
         <translation>Zum aktuellen Octave Verzeichnis gehen</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Set Octave directory</source>
         <translation>Octave Verzeichnis setzen</translation>
     </message>
     <message>
-        <location line="+2"/>
-        <source>Set Octave directroy to current browser directory</source>
-        <translation>Setze Octave Verzeichnis zu aktuellem Browser Verzeichnis</translation>
-    </message>
-    <message>
-        <location line="+7"/>
+        <location line="+9"/>
         <source>Show Home Directory</source>
         <translation>Wechsle zum Heimatverzeichnis</translation>
     </message>
     <message>
         <location line="+20"/>
         <location line="+397"/>
         <source>New File...</source>
         <translation>Neue Datei...</translation>
@@ -1007,17 +1045,17 @@ zu: </translation>
         <location line="+1"/>
         <source>Cancel</source>
         <translation>Abbrechen</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>&lt;html&gt;&lt;body&gt;
 &lt;p&gt;We hope you find Octave to be a useful tool.&lt;/p&gt;
-&lt;p&gt;If you encounter problems, there are a number of ways to get help, including commercial support options, a mailing list, a wiki, and other commnity-based support channels.
+&lt;p&gt;If you encounter problems, there are a number of ways to get help, including commercial support options, a mailing list, a wiki, and other community-based support channels.
 You can find more information about each of these by visiting &lt;a href=&quot;http://octave.org/support.html&quot;&gt;http://octave.org/support.html&lt;/a&gt; (opens in external browser).&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
         <translation>&lt;html&gt;&lt;body&gt;
 &lt;p&gt;Wir hoffen, dass Du Octave nützlich findest.&lt;/p&gt;
 &lt;p&gt;Wenn Du auf Probleme stoßen solltest, gibt es eine Reihe von Wegen um Hilfe zu bekommen. Neben der Option kommerziellen Support zu erhalten, gibt es noch eine Mailingliste, ein Wiki und andere Community Support Channels.
 Mehr über jede Möglichkeit findest Du im Internet unter &lt;a href=&quot;http://octave.org/support.html&quot;&gt;http://octave.org/support.html&lt;/a&gt; (Wird im externen Browser geöffnet).&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</translation>
     </message>
@@ -1449,17 +1487,17 @@ Klicke auf &apos;Weiter&apos; um eine Ko
     <message>
         <location line="+4"/>
         <source>Paste</source>
         <translation>Einfügen</translation>
     </message>
     <message>
         <location line="-1478"/>
         <source>Save Workspace As</source>
-        <translation>Arbeitsumgebung speichern als</translation>
+        <translation>Arbeitsumgebung speichern unter</translation>
     </message>
     <message>
         <location line="+134"/>
         <source>The release notes file &apos;%1&apos; is empty.</source>
         <translation>Die Datei &apos;%1&apos; mit Versionshinweisen ist leer.</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -1564,17 +1602,17 @@ Klicke auf &apos;Weiter&apos; um eine Ko
     <message>
         <location line="+868"/>
         <source>Load Workspace...</source>
         <translation>Lade Arbeitsumgebung...</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Save Workspace As...</source>
-        <translation>Arbeitsumgebung speichern als...</translation>
+        <translation>Arbeitsumgebung speichern unter...</translation>
     </message>
     <message>
         <location line="+38"/>
         <source>New Script</source>
         <translation>Neues Skript</translation>
     </message>
     <message>
         <location line="+115"/>
@@ -1846,22 +1884,22 @@ existiert nicht. Soll sie erzeugt werden
     <message>
         <location line="+3"/>
         <source>Octave Editor</source>
         <translation>Octave Editor</translation>
     </message>
     <message>
         <location line="+211"/>
         <source>The file %1 does not exist in the load path.  To run or debug the function you are editing, you must either change to the directory %2 or add that directory to the load path.</source>
-        <translation>Die Datei %1 befindet sich nicht im Suchpfad.  Um die momentan geöffnete Datei auszuführen oder zu debuggen, wechseln Sie entweder in das Verzeichnis %2 oder fügen Sie dieses Verzeichnis dem Suchpfad hinzu.</translation>
+        <translation>Die Datei %1 befindet sich nicht im Suchpfad.  Um die momentan geöffnete Datei auszuführen oder zu debuggen, in das Verzeichnis %2 wechseln oder dieses Verzeichnis dem Suchpfad hinzufügen.</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>The file %1 is shadowed by a file with the same name in the load path. To run or debug the function you are editing, change to the directory %2.</source>
-        <translation>Die Datei %1 wird von einer Datei gleichen Namens im Suchpfad verdeckt. Um die Funktion, die Sie gerade bearbeiten, auszuführen oder zu debuggen, wechseln Sie in das Verzeichnis %2.</translation>
+        <translation>Die Datei %1 wird von einer Datei gleichen Namens im Suchpfad verdeckt. Um die momentan geöffnete Funktion auszuführen oder zu debuggen, in das Verzeichnis %2 wechseln.</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Change Directory or Add Directory to Load Path</source>
         <translation>Verzeichnis wechseln oder zum Suchpfad hinzufügen</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -1942,23 +1980,17 @@ Die Octave-Benutzeroberfläche muss jetzt geschlossen werden.</translation>
         <translation>Titelleisten der Unterfenster</translation>
     </message>
     <message>
         <location line="+53"/>
         <source>Small</source>
         <translation>Klein</translation>
     </message>
     <message>
-        <location line="+7"/>
-        <source> Normal</source>
-        <translatorcomment>Armin: Was macht das Leerzeichen?</translatorcomment>
-        <translation type="unfinished">Normal</translation>
-    </message>
-    <message>
-        <location line="+10"/>
+        <location line="+17"/>
         <source>Large</source>
         <translation>Groß</translation>
     </message>
     <message>
         <location line="+168"/>
         <source>Custom style</source>
         <translation>Benutzerdefiniert</translation>
     </message>
@@ -1975,18 +2007,20 @@ Die Octave-Benutzeroberfläche muss jetzt geschlossen werden.</translation>
     <message>
         <location line="+151"/>
         <source>Color</source>
         <translation>Farbe</translation>
     </message>
     <message>
         <location line="+135"/>
         <source>This works well for monospaced fonts. The line is drawn at a position based on the width of a space character in the default font. It may not work very well if styles use proportional fonts or if varied font sizes or bold, italic and normal texts are used.</source>
-        <translatorcomment>Where is this string used?</translatorcomment>
-        <translation type="unfinished">Funktioniert gut mit Schriftarten fester Breite. Die Position der angezeigten Linie basiert auf der Breite eines Leerzeichens der Standardschriftart. Dies funktioniert möglicherweise nicht gut mit Stilen, die eine Schriftart variabler Breite verwenden oder wenn verschiedene Schriftgrößen oder -stile (fett/kursiv/normal) verwendet werden.</translation>
+        <translation>Funktioniert gut mit Schriftarten fester Breite. 
+Die Position der angezeigten Linie basiert auf der Breite eines Leerzeichens der Standardschriftart.
+Dies funktioniert möglicherweise nicht mit Stilen, die eine Schriftart variabler Breite verwenden 
+oder wenn unterschiedliche Schriftgrößen oder -stile (fett/kursiv/normal) verwendet werden.</translation>
     </message>
     <message>
         <location line="+19"/>
         <source>Enable Code Folding</source>
         <translation>Code-Faltung aktivieren</translation>
     </message>
     <message>
         <location line="+61"/>
@@ -2288,19 +2322,24 @@ Die Octave-Benutzeroberfläche muss jetzt geschlossen werden.</translation>
         <translation>Datum der letzten Änderung anzeigen</translation>
     </message>
     <message>
         <location line="-14"/>
         <source>Show hidden files</source>
         <translation>Versteckte Dateien anzeigen</translation>
     </message>
     <message>
-        <location line="-1377"/>
+        <location line="-1647"/>
+        <source>Normal</source>
+        <translation>Normal</translation>
+    </message>
+    <message>
+        <location line="+270"/>
         <source>These preferences are applied after any .octaverc startup files.</source>
-        <translation>Diese Einstellungen werden nach dem Laden vorhandener .octaverc-Dateien angewendet</translation>
+        <translation>Diese Einstellungen werden nach dem Laden vorhandener .octaverc-Dateien angewendet.</translation>
     </message>
     <message>
         <location line="+55"/>
         <source>Restore working directory of previous session</source>
         <translation>Arbeitsverzeichnis der vorherigen Sitzung wiederherstellen</translation>
     </message>
     <message>
         <location line="+322"/>
@@ -2326,17 +2365,17 @@ Die Octave-Benutzeroberfläche muss jetzt geschlossen werden.</translation>
     <message>
         <location line="+79"/>
         <source>File handling</source>
         <translation>Dateiverwaltung</translation>
     </message>
     <message>
         <location line="+93"/>
         <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select font, font size (as a difference from the default size), font style (&lt;b&gt;b&lt;/b&gt;old, &lt;b&gt;i&lt;/b&gt;talic, &lt;b&gt;u&lt;/b&gt;nderline), text color, and background color (for the latter, the color magenta (255,0,255) is a placeholder for the default background color).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
-        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wählen Sie Schriftart, Schriftgröße (als Differenz zur Standard-Schriftgröße), Schriftstil (&lt;b&gt;f&lt;/b&gt;ett, &lt;b&gt;k&lt;/b&gt;ursiv, &lt;b&gt;u&lt;/b&gt;nterstrichen), Textfarbe und Hintergrundfarbe (bei letzterer ist Magenta (255,0,255) ein Platzhalter für die Standard-Hintergrundfarbe).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wähle Schriftart, Schriftgröße (als Differenz zur Standard-Schriftgröße), Schriftstil (&lt;b&gt;f&lt;/b&gt;ett, &lt;b&gt;k&lt;/b&gt;ursiv, &lt;b&gt;u&lt;/b&gt;nterstrichen), Textfarbe und Hintergrundfarbe (bei letzterer ist Magenta (255,0,255) ein Platzhalter für die Standard-Hintergrundfarbe).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+341"/>
         <source>Display</source>
         <translation>Anzeige</translation>
     </message>
     <message>
         <location line="+39"/>
@@ -2406,85 +2445,85 @@ Die Octave-Benutzeroberfläche muss jetzt geschlossen werden.</translation>
     <message>
         <location line="+1714"/>
         <source>Synchronize Octave working directory with file browser</source>
         <translation>Octave-Arbeitsverzeichnis und Dateibrowser synchronisieren</translation>
     </message>
     <message>
         <location line="+132"/>
         <source>Colors for variable attributes </source>
-        <translation type="unfinished"></translation>
+        <translation>Farben für Variablen-Eigenschaften</translation>
     </message>
     <message>
         <location line="+34"/>
         <source>Shortcuts</source>
-        <translation type="unfinished"></translation>
+        <translation>Tastenkürzel</translation>
     </message>
     <message>
         <location line="+31"/>
         <source>Select this option to prevent conflicts with readline shortcuts</source>
-        <translation type="unfinished"></translation>
+        <translation>Wähle diese Option, um Konflikte mit den Tastenkürzeln von &quot;Readline&quot; zu verhindern</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Disable global shortcuts when terminal window has focus</source>
-        <translation type="unfinished"></translation>
+        <translation>Globale Tastenkürzel deaktivieren, solange Terminal-Fenster den Fokus hat</translation>
     </message>
     <message>
         <location line="+31"/>
         <location line="+28"/>
         <source>Use this set</source>
-        <translation type="unfinished"></translation>
+        <translation>Diesen Satz verwenden</translation>
     </message>
     <message>
         <location line="-21"/>
         <source>Set 1:</source>
-        <translation type="unfinished"></translation>
+        <translation>Satz 1:</translation>
     </message>
     <message>
         <location line="+7"/>
         <location line="+37"/>
         <source>Export</source>
-        <translation type="unfinished"></translation>
+        <translation>Exportieren</translation>
     </message>
     <message>
         <location line="-30"/>
         <location line="+37"/>
         <source>Import</source>
-        <translation type="unfinished"></translation>
+        <translation>Importieren</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Set 2:</source>
-        <translation type="unfinished"></translation>
+        <translation>Satz 2:</translation>
     </message>
     <message>
         <location line="+86"/>
         <source>Action</source>
-        <translation type="unfinished"></translation>
+        <translation>Aktion</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Set 1 Default</source>
-        <translation type="unfinished"></translation>
+        <translation>Satz 1 Standard</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Set 1 Actual</source>
-        <translation type="unfinished"></translation>
+        <translation>Satz 1 aktuell</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Set 2 Default</source>
-        <translation type="unfinished"></translation>
+        <translation>Satz 2 Standard</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Set 2 Actual</source>
-        <translation type="unfinished"></translation>
+        <translation>Satz 2 aktuell</translation>
     </message>
     <message>
         <location line="+68"/>
         <source>Socks5Proxy</source>
         <translation>Socks5Proxy</translation>
     </message>
     <message>
         <location line="-16"/>
@@ -2559,22 +2598,22 @@ Die Octave-Benutzeroberfläche muss jetzt geschlossen werden.</translation>
         <location line="+1"/>
         <source>u</source>
         <comment>short form for underlined</comment>
         <translation>u</translation>
     </message>
     <message>
         <location line="+497"/>
         <source>Set Octave Startup Directory</source>
-        <translation type="unfinished"></translation>
+        <translation>Octave Start-Ordner festlegen</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Set File Browser Startup Directory</source>
-        <translation type="unfinished"></translation>
+        <translation>Datei-Browser Start-Ordner wählen</translation>
     </message>
 </context>
 <context>
     <name>setup_community_news</name>
     <message>
         <location filename="../src/welcome-wizard.cc" line="+44"/>
         <source>Community News</source>
         <translation>Neuigkeiten der Community</translation>
@@ -2652,17 +2691,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+1"/>
         <source>Load Workspace</source>
         <translation>Lade Arbeitsumgebung</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Save Workspace As</source>
-        <translation>Arbeitsumgebung speichern als</translation>
+        <translation>Arbeitsumgebung speichern unter</translation>
     </message>
     <message>
         <location line="+1"/>
         <location line="+158"/>
         <source>Preferences</source>
         <translation>Einstellungen</translation>
     </message>
     <message>
@@ -2717,22 +2756,22 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+2"/>
         <source>Clear Workspace</source>
         <translation>Arbeitsumgebung löschen</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Step Over</source>
-        <translation type="unfinished"></translation>
+        <translation>Überspringen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Step Into</source>
-        <translation type="unfinished"></translation>
+        <translation>Hineinspringen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Step Out</source>
         <translation>Herausspringen</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -2808,22 +2847,22 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+2"/>
         <source>Reset Window Layout</source>
         <translation>Fensterlayout zurücksetzen</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Show Ondisk Documentation</source>
-        <translation type="unfinished"></translation>
+        <translation>Lokal installierte Dokumentation anzeigen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Show Online Documentation</source>
-        <translation type="unfinished"></translation>
+        <translation>Online-Dokumentation anzeigen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Report a Bug</source>
         <translation>Einen Programmfehler melden</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -2858,57 +2897,57 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+1"/>
         <source>Community News</source>
         <translation>Neuigkeiten der Community</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Edit Function</source>
-        <translation type="unfinished"></translation>
+        <translation>Funktion bearbeiten</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Save File</source>
-        <translation type="unfinished"></translation>
+        <translation>Datei speichern</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Save File As</source>
-        <translation type="unfinished"></translation>
+        <translation>Datei speichern unter</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>Schließen</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Close All</source>
         <translation>Alle schließen</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Close Other</source>
-        <translation type="unfinished"></translation>
+        <translation>Andere schließen</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Print</source>
-        <translation type="unfinished"></translation>
+        <translation>Drucken</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Redo</source>
-        <translation type="unfinished"></translation>
+        <translation>Wiederherstellen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Cut</source>
-        <translation type="unfinished"></translation>
+        <translation>Ausschneiden</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Find and Replace</source>
         <translation>Suchen und Ersetzen</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -2953,190 +2992,192 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+2"/>
         <source>Transpose Line</source>
         <translation>Zeilen tauschen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Completion List</source>
-        <translation type="unfinished"></translation>
+        <translation>Vervollständigungsliste anzeigen</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Comment Selection</source>
-        <translation>Auswahl auskommentieren</translation>
+        <translatorcomment>David: geändert damit identisch mit dem Editor-Menü</translatorcomment>
+        <translation>Kommentieren</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Uncomment Selection</source>
-        <translation>Auswahl entkommentieren</translation>
+        <translatorcomment>David: geändert damit identisch mit dem Editor-Menü</translatorcomment>
+        <translation>Kommentar entfernen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Uppercase Selection</source>
-        <translation type="unfinished"></translation>
+        <translation>Auswahl zu Großbuchstaben</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Lowercase Selection</source>
-        <translation type="unfinished"></translation>
+        <translation>Auswahl zu Kleinbuchstaben</translation>
     </message>
     <message>
         <location line="+4"/>
         <location line="+5"/>
         <source>Indent Selection</source>
-        <translation type="unfinished"></translation>
+        <translation>Einzug erhöhen</translation>
     </message>
     <message>
         <location line="-3"/>
         <location line="+5"/>
         <source>Unindent Selection</source>
-        <translation type="unfinished"></translation>
+        <translation>Einzug verringern</translation>
     </message>
     <message>
         <location line="+4"/>
-        <source>Convert Line Ednings to Windows</source>
-        <translation type="unfinished"></translation>
+        <source>Convert Line Endings to Windows</source>
+        <translation>Zeilenenden für Windows umwandeln</translation>
     </message>
     <message>
         <location line="+2"/>
-        <source>Convert Line Ednings to Unix</source>
-        <translation type="unfinished"></translation>
+        <source>Convert Line Endings to Unix</source>
+        <translation>Zeilenenden für Linux umwandeln</translation>
     </message>
     <message>
         <location line="+2"/>
-        <source>Convert Line Ednings to Mac</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <location line="+3"/>
+        <source>Convert Line Endings to Mac</source>
+        <translation>Zeilenenden für Mac umwandeln</translation>
+    </message>
+    <message>
+        <location line="+25"/>
+        <source>Show Whitespace Characters</source>
+        <translation>Leerzeichen anzeigen</translation>
+    </message>
+    <message>
+        <location line="-22"/>
         <source>Goto Line</source>
-        <translation type="unfinished"></translation>
+        <translation>Gehe zu Zeile</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Move to Matching Brace</source>
         <translation>Zur schließenden Klammer springen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Select to Matching Brace</source>
         <translation>Auswahl bis zur schließenden Klammer</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Toggle Bookmark</source>
-        <translation type="unfinished"></translation>
+        <translation>Lesezeichen umschalten</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Next Bookmark</source>
-        <translation type="unfinished"></translation>
+        <translation>Nächstes Lesezeichen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Previous Bookmark</source>
-        <translation type="unfinished"></translation>
+        <translation>Vorheriges Lesezeichen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Remove All Bookmark</source>
-        <translation type="unfinished"></translation>
+        <translation>Alle Lesezeichen entfernen</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Styles Preferences</source>
-        <translation type="unfinished"></translation>
+        <translation>Stil-Einstellungen</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Show Line Numbers</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <location line="+2"/>
-        <source>Show White Spaces</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <location line="+2"/>
+        <translation>Zeilennummern anzeigen</translation>
+    </message>
+    <message>
+        <location line="+4"/>
         <source>Show Line Endings</source>
-        <translation type="unfinished"></translation>
+        <translation>Zeilenenden anzeigen</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Show Indentation Guides</source>
-        <translation type="unfinished"></translation>
+        <translation>Einrückungshinweise anzeigen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Show Long Line Marker</source>
-        <translation type="unfinished"></translation>
+        <translation>Markierung langer Zeilen anzeigen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Zoom In</source>
         <translation>Vergrößern</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Zoom Out</source>
         <translation>Verkleinern</translation>
     </message>
     <message>
         <location line="+2"/>
         <location line="+3"/>
         <source>Zoom Normal</source>
-        <translation type="unfinished"></translation>
+        <translation>Normale Größe</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Toggle Breakpoint</source>
-        <translation type="unfinished"></translation>
+        <translation>Lesezeichen umschalten</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Next Breakpoint</source>
-        <translation type="unfinished"></translation>
+        <translation>Nächster Haltepunkt</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Previous Breakpoint</source>
-        <translation type="unfinished"></translation>
+        <translation>Vorheriger Haltepunkt</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Remove All Breakpoints</source>
-        <translation type="unfinished"></translation>
+        <translation>Alle Haltepunkte entfernen</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Run File</source>
-        <translation type="unfinished"></translation>
+        <translation>Datei ausführen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Run Selection</source>
-        <translation type="unfinished"></translation>
+        <translation>Auswahl ausführen</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Help on Keyword</source>
-        <translation type="unfinished"></translation>
+        <translation>Hilfe zum Schlüsselwort</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Document on Keyword</source>
-        <translation type="unfinished"></translation>
+        <translation>Dokumentation zum Schlüsselwort</translation>
     </message>
     <message>
         <location line="+42"/>
         <source>Main</source>
-        <translation type="unfinished"></translation>
+        <translation>Hauptprogramm</translation>
     </message>
     <message>
         <location line="+3"/>
         <location line="+23"/>
         <source>File</source>
         <translation>Datei</translation>
     </message>
     <message>
@@ -3144,17 +3185,17 @@ a:link { text-decoration: underline; col
         <location line="+23"/>
         <source>Edit</source>
         <translation>Bearbeiten</translation>
     </message>
     <message>
         <location line="-21"/>
         <location line="+25"/>
         <source>Debug</source>
-        <translation type="unfinished"></translation>
+        <translation>Debuggen</translation>
     </message>
     <message>
         <location line="-23"/>
         <source>Window</source>
         <translation>Fenster</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -3165,87 +3206,91 @@ a:link { text-decoration: underline; col
     <message>
         <location line="-23"/>
         <source>News</source>
         <translation>Neuigkeiten</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>View</source>
-        <translation type="unfinished"></translation>
+        <translation>Ansicht</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Run</source>
         <translation>Ausführen</translation>
     </message>
     <message>
         <location line="+124"/>
         <source>Enter new Shortcut for Set %1</source>
-        <translation type="unfinished"></translation>
+        <translation>Neues Tastenkürzel für Satz %1 eingeben</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Apply the desired shortcut or click on the right button to reset the shortcut to its default.</source>
-        <translation type="unfinished"></translation>
+        <translation>Das gewünschte Tastenkürzel eingeben oder auf die Schaltfläche rechts klicken, um das Tastenkürzel auf seinen Standardwert zurückzusetzen.</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Enter shortcut directly by performing it</source>
-        <translation type="unfinished"></translation>
+        <translation>Tastenkürzel direkt durch seine Ausführung eingeben</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Actual shortcut</source>
-        <translation type="unfinished"></translation>
+        <translation>Aktives Tastenkürzel</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Default shortcut</source>
-        <translation type="unfinished"></translation>
+        <translation>Standard-Tastenkürzel</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Set to default</source>
-        <translation type="unfinished"></translation>
+        <translation>Auf Standardwert setzen</translation>
     </message>
     <message>
         <location line="+53"/>
         <source>Double Shortcut</source>
-        <translation type="unfinished"></translation>
+        <translation>Doppeltes Tastenkürzel</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>The chosen shortcut
   &quot;%1&quot;
 is already used for the action
   &quot;%2&quot;.
 Do you want to use the shortcut anyhow removing it from the previous action?</source>
-        <translation type="unfinished"></translation>
+        <translation>Das gewählte Tastenkürzel
+  &quot;%1&quot;&quot;
+wird bereits für die Aktion
+  &quot;%2&quot;&quot;
+verwendet. Soll das Tastenkürzel trotzdem verwendet werden und damit von der vorgenannten Aktion entfernt werden?</translation>
     </message>
     <message>
         <location line="+71"/>
         <source>Import shortcut set %1 from file ...</source>
-        <translation type="unfinished"></translation>
+        <translation>Tastenkürzel-Satz %1 aus Datei importieren ...</translation>
     </message>
     <message>
         <location line="+1"/>
         <location line="+6"/>
         <source>Octave Shortcut Files (*.osc);;All Files (*)</source>
-        <translation type="unfinished"></translation>
+        <translation>Octave Tastenkürzel-Dateien (*.osc);;Alle Dateien (*)</translation>
     </message>
     <message>
         <location line="-1"/>
         <source>Export shortcut set %1 into file ...</source>
-        <translation type="unfinished"></translation>
+        <translation>Tastenkürzel-Satz %1 in Datei exportieren ...</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>Failed to open %1 as octave shortcut file</source>
-        <translation type="unfinished"></translation>
+        <translation>Konnte %1 nicht als Octave-Tastenkürzel-Datei öffnen</translation>
     </message>
 </context>
 <context>
     <name>terminal_dock_widget</name>
     <message>
         <location filename="../src/terminal-dock-widget.cc" line="+38"/>
         <source>Command Window</source>
         <translation>Befehlsfenster</translation>
@@ -3266,17 +3311,17 @@ Do you want to use the shortcut anyhow r
     <message>
         <location line="+18"/>
         <source>Error</source>
         <translation>Fehler</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>The info file&lt;p&gt;%1&lt;p&gt;or compressed versions do not exist</source>
-        <translation type="unfinished"></translation>
+        <translation>Die Info-Datei &lt;p&gt;%1&lt;p&gt;existiert nicht, auch keine komprimierte Version</translation>
     </message>
 </context>
 <context>
     <name>welcome_wizard</name>
     <message>
         <location filename="../src/welcome-wizard.cc" line="+135"/>
         <source>Welcome to GNU Octave</source>
         <translation>Willkommen bei GNU Octave</translation>
